ingest/src/esoh/ingest/datastore.py

```diff
@@ -98,6 +98,17 @@ def ingest(self, msg: str) -> None:
             elif i in msg["properties"]["content"]:
                 setattr(ts_metadata, i, msg["properties"]["content"][i])
 
+        # Special case for metadata_id -> time_series_id. This translation is required for naming
+        # consistency with the terminology used internally in the datastore where a 'time series'
+        # represents metadata that usually don't change with obs time. This metadata is represented
+        # in the TSMetadata message in datastore.proto and used for defining request- and response
+        # messages for several methods.
```

Comment:
Okey. Add a TODO, this should not be renamed here, but instead changed in the e-soh-message spec.

Reply:
Will do.